.github/workflows/build-pr-development.yml

```diff
@@ -19,6 +19,13 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
+      - name: Check Wildcards
+        run: |
+          if [ $(git grep import | grep '\*;' | grep -E -v '(.mvn/|README.md|CONTRIBUTING.md)' | wc -l) != 0 ]; then
+            echo "Wildcard found!"
```

Comment:
This is not a helpful error message. `Please don't use star imports in your Java files:` would be better.

Reply:
PR was updated as requested, resolving the conversation